cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/Outlet.java

```diff
@@ -30,21 +30,48 @@
 import cwms.cda.formatters.Formats;
 import cwms.cda.formatters.annotations.FormattableWith;
 import cwms.cda.formatters.json.JsonV1;
+import java.util.ArrayList;
+import java.util.List;
 
 @FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
 @JsonDeserialize(builder = Outlet.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
 @JsonPropertyOrder({ "projectId", "location" })
 public class Outlet extends ProjectStructure {
+    private final List<CompoundOutletRecord> compoundOutletRecords = new ArrayList<>();
```

Comment:
List doesn't protect against duplicate or clashes and implies order (and I think order is a graph and not an array?). What about a TreeSet where comparable is based on the outlet id?

Reply:
We don't want to protect against duplicates or clashes.  If the db or the user gives us bad data, we shouldn't obviate that by removing the bad data.
As for the ordering, I don't believe the ordering matters from the database, and List is just the first collection that comes to mind.  I could use a Set, but we wouldn't want to do hashCode/equals on the object because of my reasoning above, so I'm not sure what benefit that will provide at this layer.

Reply:
@rma-psmorris, what are your thoughts on this?

Reply:
@RyanM-RMA -- Why does the Outlet.java know about CompoundOutletRecords? CompoundOutlets are a very specific type and to include that dependency in all outlets whether controlled or uncontrolled seems like a bad dependency.

Reply:
I added that based on what the HCDA's JDBC code does.